src/app.scss

```diff
@@ -33,13 +33,13 @@
 }
 
 .pf-v5-c-page__main {
+    gap: var(--pf-v5-global--spacer--md);
     display: grid;
     grid-template:
         "topbar topbar topbar topbar" auto
         ". content sidebar ." 1fr
         ". . . ." 0
         / 0 1fr 18rem 0;
-    gap: var(--pf-v5-global--spacer--md);
```

Comment:
Was this one necessary? Just wondering...

Reply:
I suppose not, as it's not really nested, but :man_shrugging: